gh-129044: Update glossary entry for 'loader' to reflect current import system (#129073)
python/cpython: Doc/glossary.rst

      processed.

   loader

>       An object that loads a module. It must define the
>       :meth:`!exec_module` and :meth:`!create_module` methods
>       to implement the :class:`~importlib.abc.Loader` interface.
>       A loader is typically returned by a :term:`finder`. See also:

Reviewer: @nikalinov note you can use the "Commit suggestion" button in the GH interface to apply a reviwer's suggestions. This keeps newlines to reduce the diff for any future changes.
```suggestion
      An object that loads a module.
      It must define the :meth:`!exec_module` and :meth:`!create_module` methods
      to implement the :class:`~importlib.abc.Loader` interface.
      A loader is typically returned by a :term:`finder`.
      See also:
```

   loader

>       An object that loads a module. It must define a method named

Reviewer: ```suggestion
      An object that loads a module.
```

>       :meth:`exec_module`, along with :meth:`create_module`.
>       The :meth:`load_module` method, which was used in earlier
>       versions of Python, has been deprecated since version 3.4
>       and should no longer be implemented in new code. A loader is
>       typically returned by a :term:`finder`. See also:

Reviewer: ```suggestion
      It must define the :meth:`!exec_module` and :meth:`!create_module`
      methods to implement the :class:`~importlib.abc.Loader` interface.
      A loader is typically returned by a :term:`finder`.
      See also:
```
